allauth/socialaccount/providers/dingtalk/provider.py

```diff
@@ -0,0 +1,30 @@
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class DingtalkAccount(ProviderAccount):
+    def get_avatar_url(self):
+        return self.account.extra_data.get("avatarUrl")
+
+    def to_str(self):
+        return self.account.extra_data.get(
+            "nick", super(DingtalkAccount, self).to_str()
+        )
+
+
+class DingtalkProvider(OAuth2Provider):
+    id = "dingtalk"
+    name = "Dingtalk"
```

Comment:
**Issue**: The official name for this provider seems to be "DingTalk" -- can you please make sure that is used in allauth as well? That holds for the classes as well, e.g. `DingTalkAccount`

Reply:
ok, the `DingTalk` is right.